.github/workflows/check-manifest.yml

```diff
@@ -4,12 +4,16 @@ on: pull_request
 
 jobs:
   validate:
-    name: Validate JSON
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 15.x
-      - run: npx ajv-cli validate -s manifest.schema.json -d manifest.json
+          node-version: 16.x
+      - name: Validate Manifest
+        working-directory: tools/validate-manifest
+        run: |
+          npm ci
+          npm run build
+          npm run test
+          node . validate ../../manifest.json
```

Comment:
i'm a little confused by the syntax here with `node . validate`.

could you make validate a script in the package.json file so the command is smthn like `npm run validate ../..manifest.json`?

Reply:
Done, though I'm not sure it's actually that much clearer either way. Oh well!